glean_parser/schemas/baseline.1.schema.json

```diff
@@ -0,0 +1,424 @@
+{
```

Comment:
I think I have strong feelings about this, given that I messed up quite a few time Desktop Telemetry due to out of sync dependencies (that *I* forgot to update :) ).

What about we keep this generic, drop the schema off this repo, pass in a path to the schema when invoking the command?